hw1/02-palindrome.js

```diff
@@ -1,3 +1,53 @@
-const elem = document.querySelector('input');
+/**
+ * Checks if a string is a positive number or not.
+ *
+ * Examples:
+ *
+ * '0'    : false \
+ * 'abc'  : false \
+ * '-1'   : false \
+ * '001'  : true \
+ * '1.23' : true \
+ * 'Infinity' : true
+ */
+const isPos = function isPositiveNumber(str) {
+  if (!Number.isNaN(str) && Number(str) > 0) {
+    return true;
+  }
+  return false;
+};
 
-elem.addEventListener('input', handleInput);
+/**
+ * Checks if a string is a palindrome or not.
+ */
+const isPal = function isPalindrome(str) {
```

Comment:
Seems like a good working function I checked your Equality operators and array use against the Airbnb guide and looks good here. good job determining appropriate "let" vs "const" use